### ISSUE TITLE
Failure to copy build.log file

### ISSUE THREAD

**Issue by mohatt** [mdkholy/dashbrew#18]
Failure to copy build.log file after successful build.

```
[Error] Failed to copy "/opt/phpbrew/build/php-5.3.29/build.log" because file does not exist.
```

**Comment by mdkholy** [comment]
Fixed in #d1d111036527b1381b0f0e9a99933d351ba9f04c

**Comment by mohatt** [comment]
Fixed in d1d111036527b1381b0f0e9a99933d351ba9f04c
